critiquebrainz/db/review_test.py

```diff
@@ -196,7 +196,7 @@ def test_list_reviews(self):
         self.assertEqual(count, 1)
         self.assertEqual(len(reviews), 1)
 
-        reviews, count = db_review.list_reviews(sort="created")
+        reviews, count = db_review.list_reviews(sort="")
```

Comment:
I think you should put `published_on` as an argument here. This would test if the reviews are fetched successfully if ordered by publish date.

Reply:
sorry, i already have the "published_on" argument set up, i think i was in the middle of adding it here and got distracted, and ended up leaving it blank

Reply:
No problem :)